src/ontology/mondo-edit.obo

```diff
@@ -165662,6 +165652,7 @@ xref: SCTID:721307000 {source="MONDO:equivalentTo"}
 xref: UMLS:C1834582 {source="MEDGEN:331782", source="MONDO:equivalentTo", source="MONDO:MEDGEN"}
 is_a: MONDO:0020076 {source="DOID:0060888", source="Orphanet:420611"} ! myeloproliferative neoplasm
 relationship: disease_arises_from_feature MONDO:0008608 ! Down syndrome
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4170 {source="OMIM:159595"} ! GATA1
```

Comment:
Same comment as below. 
If this annotation is expected as per our rules and the file we use, then all is well. 
It should however be removed manually as it is not correct according to the website.

Reply:
Ok, I see this is for MONDO:0008040 'transient myeloproliferative syndrome' and that the OMIM [page](https://omim.org/entry/159595) shows this with a mapping key of 2 and should not have been added. @joeflack4 can you sort out this bug?